capstone/src/main/java/com/google/sps/data/WordcloudData.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+/** Data for associations wordcloud */
+public final class WordcloudData {
```

Comment:
Is "word cloud" two words and should be camel cased like so (ex: `WordCloudData`)?

Reply:
I'm not actually sure, I've seen it written as both one and two words. I should standardize it between the classes though.

Reply:
Swapped it to two words camel case!